test/e2e/config/ibmcloud-e2e.yaml

```diff
@@ -0,0 +1,58 @@
+managementClusterName: capi-ibmcloud-e2e
+
+providers:
+  - name: cluster-api
+    type: CoreProvider
+    versions:
+    - name: v1.0.2
+      value: https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.0.2/core-components.yaml
+      type: url
+      files:
+      - sourcePath: "../data/shared/metadata.yaml"
+  - name: kubeadm
+    type: BootstrapProvider
+    versions:
+    - name: v1.0.2
+      value: https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.0.2/bootstrap-components.yaml
+      type: url
+      files:
+      - sourcePath: "../data/shared/metadata.yaml"
+  - name: kubeadm
+    type: ControlPlaneProvider
+    versions:
+    - name: v1.0.2
+      value: https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.0.2/control-plane-components.yaml
+      type: url
+      files:
+      - sourcePath: "../data/shared/metadata.yaml"
+  - name: ibmcloud
+    type: InfrastructureProvider
+    versions:
+    - name: v0.2.0
+      value: "${PWD}/config/default"
+      files:
+      - sourcePath: "${PWD}/metadata.yaml"
+        targetName: "metadata.yaml"
+      - sourcePath: "${PWD}/templates/cluster-template-powervs.yaml"
+        targetName: "cluster-template-powervs.yaml"
+      - sourcePath: "${PWD}/templates/cluster-template.yaml"
+        targetName: "cluster-template-vpc.yaml"
+
+variables:
+  KUBERNETES_VERSION: "${KUBERNETES_VERSION:-v1.22.4}"
+  # Cluster Addons
+  CNI: "${PWD}/test/e2e/data/cni/calico/calico.yaml"
```

Comment:
Above you have used `../data`, wondering same can be used here as well?

Reply:
This might not work.

Reply:
but other places in the file got that syntax?!